docs/UserGuide.md

```diff
@@ -76,18 +76,32 @@ List all contacts in the address book.
 
 Format: `list`
 
-### Locating persons by name/tags: `find`
+### Locating persons by keywords: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds persons whose contact details contain any of the given keywords based on the
+prefix specified.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find [PREFIX]/KEYWORD [MORE [PREFIX]/KEYWORD]...`
 
 * The search is case-insensitive e.g. `hans` will match `Hans`
-* The order of the keywords does not matter e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search) e.g. `Hans Bo` will return `Hans Gruber`
-  , `Bo Yang`
+* The search will filter by the `PREFIX` provided, e.g. `n/` searches through the
```

Comment:
User guide is up-to-date and v1.3 progress is advancing well. Some touch-ups and bug fixes will do before the feature freeze in v1.4